frontend/components/OurStoryTimeline.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from "react";
+import styles from "../styles/OurStoryTimeline.module.css";
+import Timeline from "../components/Timeline";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import {
+  marks,
+  valueToYear,
+  pageToValue,
+  timelinePhoto,
+} from "../data/ourStoryData";
+import Image from "next/image";
+
+export default function OppositeContentTimeline() {
+  const [year, setYear] = useState(valueToYear[250]);
+  const [loading, setLoadingCircle] = useState(true);
+  // set the year for the events timeline
+  const handleYear = (newYear) => {
+    setYear(newYear);
+  };
+
+  // Show loading circle for 250 ms when moving between years
+  async function loadingCircle() {
+    setLoadingCircle(true);
+    await new Promise((resolve) => setTimeout(resolve, 250));
+    setLoadingCircle(false);
+  }
+
+  useEffect(() => {
+    loadingCircle();
+  }, [year]);
+
+  return (
+    <>
+      <Timeline
+        id={styles.timelineStyle}
+        page={"ourStory"}
+        step={20}
+        valueToYear={valueToYear}
+        marks={marks}
+        updateYear={handleYear}
+      />
+
+      <div className={styles.loadingContainer}>
+        {loading && (
+          <CircularProgress
+            variant="indeterminate"
+            size={50}
+            thickness={5}
+            id={styles.timelineLoading}
+          />
+        )}
+      </div>
+      {!loading && (
+        <div className={styles.timelineCard}>
+          <div className={styles.timelineCardImageContainer}>
+            <Image
+              src={`/our-story/timelinePhotos/${year}.jpg`}

```

Comment:
Weird issues with the our story images on my end. Check that all images laod correctly for each year and are the right dimensions for the container... 
![image](https://user-images.githubusercontent.com/80564580/177752401-41449db5-a516-41d6-bb69-883bc9fe537b.png)

Reply:
Heyy, i was wondering which device created this weird dimension, because for me, the image looks alright on all the devices on the chrome developers tool